app/controllers/users_controller.rb

```diff
@@ -15,4 +15,8 @@ def update
       render :edit
     end
   end
+
+  def bids_placed
```

Comment:
Yeah I think the URL is fine, but the actual rails route could be more REST-ful. (so you'd map `bids_placed` to `bids#index` or `account/bids#index` in config/routes.rb)